### PR TITLE
added unit tests

### PR DESCRIPTION
**Pull request by AlenaFedo**

### CODE REVIEW

File: src/test/java/shop/CartTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        JsonParser parser = new JsonParser();
+        cart = parser.readFromFile(new File("src/main/resources/andrew-cart.json"));
+        item = new RealItem();
+        item.setPrice(10);
+        cart.addRealItem(item);
+    }
+
+    @Test
+    void testCalculationWhileAddingItem() {
+        assertEquals(25, cart.getTotalPrice());
```
Comment (LikBez123) [inline src/test/java/shop/CartTest.java]: check expected result

File: src/test/java/shop/CartTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    @Test
+    void testCalculationWhileDeletingItem() {
+        cart.deleteRealItem(item);
+        assertEquals(25, cart.getTotalPrice());
```
Comment (LikBez123) [inline src/test/java/shop/CartTest.java]: check expected result

File: src/test/java/shop/VirtualItemTests.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+class VirtualItemTests {
+    private VirtualItem item;
+
+    @BeforeEach
+    public void init() {
+        item = new VirtualItem();
+        item.setSizeOnDisk(85000);
+    }
+    @Test
```
Comment (LikBez123) [inline src/test/java/shop/VirtualItemTests.java:13]: separate tests with an empty line

File: src/test/java/shop/VirtualItemTests.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package shop;
+
+import org.junit.jupiter.api.*;
+
+
+
```
Comment (LikBez123) [inline src/test/java/shop/VirtualItemTests.java]: remove extra lines

File: src/test/java/shop/VirtualItemTests.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    @BeforeEach
+    public void init() {
+        item = new VirtualItem();
+        item.setSizeOnDisk(85000);
+    }
+    @Test
+    void testVirtualItem() {
+        Assertions.assertTrue(item.toString().contains("Size on disk: 85000"));
+    }
+
```
Comment (LikBez123) [inline src/test/java/shop/VirtualItemTests.java:17]: remove extra line

**Review by LikBez123 — CHANGES_REQUESTED** [review]

File: src/test/java/shop/VirtualItemTests.java (hunk 3 above)
Comment (AlenaFedo) [reply src/test/java/shop/VirtualItemTests.java:13]: Done

File: src/test/java/shop/VirtualItemTests.java (hunk 5 above)
Comment (AlenaFedo) [reply src/test/java/shop/VirtualItemTests.java:17]: Done

File: src/test/java/shop/CartTest.java (hunk 2 above)
Comment (AlenaFedo) [reply src/test/java/shop/CartTest.java]: Done

**Review by LikBez123 — APPROVED** [review]

**Pull request merged**